script/Arcueid/Day 8/08_00_ARC08_6B.txt

```diff
@@ -54,7 +54,7 @@ Perhaps it's the brilliant white of the light leaking from the rooms.
 -- Page 355, Offset 20556.
 -- たった今マンションに戻ってきた女性の体に渦巻く、死の線の多さだった。
 Or maybe it's the woman just returning to the apartment,
- on whose body the lines of death draw a dense, whirling maelstrom.
+ on whose body lines of death draw a dense, whirling maelstrom.
```

Comment:
prefer to keep 'the' for both cases

Reply:
Hmm sorry but that suggestion isn't grammatical. I'm with Yusuke here, these sentences are fine as -is.